### PR TITLE
PAPAY-6123 | Proxysql support for AWS Aurora

### PR DESCRIPTION
**Pull request by olivernaaris**
Add environment variable for proxysql to support AWS Aurora based on feature flag

### CODE REVIEW

File: stable/proxysql/templates/deployment.yaml [hunk 1]
```diff
@@ -51,6 +51,9 @@ spec:
               containerPort: 6070
               protocol: TCP
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          - name: USE_AWS_AURORA
```
Comment (kahootali) [inline stable/proxysql/templates/deployment.yaml:55]: isn't it better to copy all envs from values?

Comment (olivernaaris) [reply stable/proxysql/templates/deployment.yaml:55]: I wanted to to keep that specific variable separate from other environment variable

Comment (olivernaaris) [reply stable/proxysql/templates/deployment.yaml:55]: I can add separate env variable block that can be templated. But for sure I want to keep this variable separate from that

**Comment by kahootali** [comment]
need to increase Chart version as well

**Pull request merged**